src/fluree/server/consensus/subscriptions.clj

```diff
@@ -3,14 +3,12 @@
             [fluree.db.util.json :as json]
             [fluree.db.util.log :as log]
             [ring.adapter.jetty9.websocket :as ws])
-  (:import (java.io IOException)
+  (:import (java.io Closeable IOException)
            (java.nio.channels ClosedChannelException)))
 
-;; structure of subscriptions atom:
-;{:subs    {"id" {:chan   port ;; <- core.async channel
-;                 :ledgers {"some/ledger" {}} ;; <- map where each key is a ledger-id subscribed to
-;                 :did     nil}}
-; :closed? false}
+(defprotocol Publicizer
```

Comment:
This is minor, but I feel like this "publishes" rather than "publicizes" and would maybe be better with the name `Publisher`, `publish-commit`, and `publish-new-ledger`. I don't insist on it, though : )

Reply:
We already have a `Publisher` protocol that the name services implement, so I didn't want to reuse that term.